src/WorkerPool.js

```diff
@@ -94,6 +112,19 @@ class PoolWorker {
   onWorkerMessage(message, finalCallback) {
     const { type, id } = message;
     switch (type) {
+      case 'end': {
+        var err = new Error('worker terminated');
+        for (var id in this.jobs) {
```

Comment:
This also iterates inherited jobs, use Object.keys()

Reply:
Done